cspell.yaml

```
@@ -60,6 +60,9 @@ words:
   - vswhere
   - westus
   - xplat
+  - segmentof
+  - createsorreplacesresource
+  - createsorudatesresource
```

Is there a typo here or is this spelled wrong in the code somewhere?

```suggestion
  - createsorupdatesresource
```